src/content/docs/en/guides/integrations-guide/vercel.mdx

```diff
@@ -277,7 +277,11 @@ export default defineConfig({
             // A secret random string that you create.
             bypassToken: "005556d774a8",
             // Paths that will always be served fresh.
-            exclude: [ "/api/invalidate", "/posts/[...slug]" ]
+            exclude: [
+              '/preview', 
+              '/auth/[page]',
+              /^\/api\/.+/ // Regular expressions supported since v8.1.0
```

Comment:
```suggestion
              /^\/api\/.+/ // Regular expressions supported since @astrojs/vercel@8.1.0
```
I think we can get away without back ticks because this is a code block anyway and won't render as inline code? Not opposed either way.